pkg/controller/intrusiondetection/intrusiondetection_controller.go

```diff
@@ -471,7 +471,15 @@ func (r *ReconcileIntrusionDetection) Reconcile(ctx context.Context, request rec
 		BindNamespaces:               namespaces,
 		Tenant:                       tenant,
 		ExternalElastic:              r.elasticExternal,
-	}
+		SyslogForwardingIsEnabled:    syslogForwardingIsEnabled(lc),
+	}
+	intrusionDetectionNamespaceComponent := render.IntrusionDetectionNamespaceComponent(&render.IntrusionDetectionNamespaceConfiguration{
+		KubernetesProvider:        network.KubernetesProvider,
+		Namespace:                 helper.InstallNamespace(),
+		Tenant:                    tenant,
+		HasNoLicense:              hasNoLicense,
+		SyslogForwardingIsEnabled: syslogForwardingIsEnabled(lc),
```

Comment:
Two nits:
 - I don't really like negative booleans, like HasNoLicense, but I see that this was already done before you changed this file.
 - You pass tenant, but not logcollector. In stead you pass SyslogForwardingIsEnabled. It seems a bit inconsistent and maybe the syslogForwardingIsEnabled func could be moved to the render file in stead. I am guessing that you did it because the boolean is evaluated more than once?
 
 I'm approving the PR regardless.

Reply:
`SyslogForwardingIsEnabled` is perform 3-4 times. Might be a good change to move it to a variable.